v4-cokapi/cokapi.js

```diff
@@ -37,7 +37,7 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 var IS_DEBUG = false;
 
-var PRODUCTION_PORT = 3000;
+var PRODUCTION_PORT = 3001;
```

Comment:
Ubuntu 16.04 has a system process running by default in port 3000. So I couldn't run it unless I changed the port. As we maybe deploying in some similar system maybe it caused problems. However now that I see that it would be better to put the port as a console parameter (With the issue of having to change the fronted) What do you think